.github/workflows/build-and-test.yml

```diff
@@ -174,22 +175,13 @@ jobs:
           path: |
             application.log
 
-  notify-slack:
+  notify-slack-on-failure:
     needs: [ build, jib, unit-tests, integration-tests ]
-    runs-on: ubuntu-latest
-
-    if: failure() && github.ref == 'refs/heads/dev'
-
-    steps:
-      - name: Notify slack on failure
-        uses: 8398a7/action-slack@v3
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.POLICY_INT_WEBHOOK_URL }}
```

Comment:
Will you delete this secret once the PR merges?

Reply:
Yes, thank you for the reminder!

Reply:
Secret has been deleted ✅